FirebaseFunctions/Sources/FunctionsError.swift

```diff
@@ -224,6 +225,8 @@ struct FunctionsError: CustomNSError {
 
     var userInfo = [String: Any]()
     userInfo[NSLocalizedDescriptionKey] = description
+    userInfo["region"] = region
+    userInfo["url"] = url.absoluteString
```

Comment:
I think it could be slightly better to store it as a URL, that way readers could get use URL instance methods right away without needing to cast the dictionary's value.

example:
```suggestion
    userInfo["url"] = url
```

Reply:
Yes, good catch. I've changed it.